libtransact/src/protos.rs

```diff
@@ -44,6 +47,9 @@ impl StdError for ProtoConversionError {
 impl std::fmt::Display for ProtoConversionError {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         match *self {
+            ProtoConversionError::DeserializationError(ref s) => {
+                write!(f, "DeserializationError: {}", s)
```

Comment:
These are effectively the equivalent of the `Debug` output.  These should be more of a human-readable display string.

Reply:
Added commit to update these error messages